contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_pair_values_to_have_diff.py

```diff
@@ -0,0 +1,176 @@
+"""
+This is an example of a Custom QueryExpectation.
+For detailed information on QueryExpectations, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations
+"""
+
+from typing import Optional, Union
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.exceptions.exceptions import (
+    InvalidExpectationConfigurationError,
+)
+from great_expectations.execution_engine import ExecutionEngine
+from great_expectations.expectations.expectation import (
+    ExpectationValidationResult,
+    QueryExpectation,
+)
+from great_expectations.expectations.metrics.import_manager import (
+    pyspark_sql_Row,
+    sqlalchemy_engine_Row,
+)
+
+
+class ExpectQueriedColumnPairValuesToHaveDiff(QueryExpectation):
+    """Expect the frequency of occurrences of a specified value in a queried column to be at least <mostly> percent of values in that column."""
+
+    metric_dependencies = ("query.column_pair",)
+
+    query = """
+            SELECT {column_A} - {column_B} as diff
+            FROM {active_batch}
+            """
+
+    success_keys = ("column_A", "column_B", "diff", "mostly", "query", "strict")
+
+    domain_keys = ("batch_id", "row_condition", "condition_parser")
+
+    default_kwarg_values = {
+        "result_format": "BASIC",
+        "include_config": True,
+        "catch_exceptions": False,
+        "meta": None,
+        "column_A": None,
+        "column_B": None,
+        "diff": 0,
+        "mostly": 1,
+        "strict": True,
+        "query": query,
+    }
+
+    def validate_configuration(
+        self, configuration: Optional[ExpectationConfiguration]
+    ) -> None:
+        super().validate_configuration(configuration)
+        diff = configuration["kwargs"].get("diff")
+        mostly = configuration["kwargs"].get("mostly")
+
+        try:
+            assert diff is not None, "'diff' must be specified"
+            assert isinstance(diff, (int, float)), "`diff` must be a valid float or int"
+            assert (
+                isinstance(mostly, (int, float)) and 0 < mostly <= 1
+            ), "'mostly' must be 1, a float between 0 and 1"
+        except AssertionError as e:
+            raise InvalidExpectationConfigurationError(str(e))
+
+    def _validate(
+        self,
+        configuration: ExpectationConfiguration,
+        metrics: dict,
+        runtime_configuration: dict = None,
+        execution_engine: ExecutionEngine = None,
+    ) -> Union[ExpectationValidationResult, dict]:
+        diff: Union[float, int] = configuration["kwargs"].get("diff")
+        mostly: str = configuration["kwargs"].get("mostly")
+        strict: bool = configuration["kwargs"].get("strict")
+        query_result: Union[sqlalchemy_engine_Row, pyspark_sql_Row] = metrics.get(
+            "query.column_pair"
+        )
+
+        if mostly != 1 and strict is True:
```

Comment:
@austiezr Sorry, I was hoping that the value of `mostly` being `1` will not need to be handled in any special way.  I believe that it should be possible to let everything be handled by `>` and `>=`, regardless of the value of `mostly`.  Do you agree?  Happy to discuss.  Thanks!

Reply:
Hey @alexsherstinsky ! Agree; realized I was still working with a `strict` parameter held over from another template. Simplified now, tests pass, should be good to go!